GitHubPolicy.md

```diff
@@ -0,0 +1,177 @@
+    * [Create a PR]()
+    * [Merge]()
+    * [Rebase]()
+    * [Squash and Merge]()
+    * [Translations]()
+7. [Github Issue Labels](#github-issue-labels)
+    
+
+## Overview
+
+Digital Democracy's codebase is maintained through Github Version Control. Pull requests can be made if you would like to contribute to our codebase. Before creating a pull request please refer to our [contributing doc]().
+
+Please refer to
+## Glossary of Term
+* PR
+    > Acronym for "Pull Request"
+* Rebase 
+    > This is the process of reording the commit history, so the commits on the PR show up at the head of the commit history. In otherwords, we are changing the base of the commit history, so that the history remains linear.
+
+* Merge
+    > This is the process of combining the commit history so that commits of the PR just merge with the commits of the base branch.
+
+* Version Control 
+    > This is the process of tracking and managing edits to a codebase. 
+
+* Pull Request
+
+## Best Practices
+[I am a contibutor](#for-contributors)
+
+[I am reviewing a PR](#reviwing-pr)
+
+### **For contributors:**
+* Follow the template.
+    > When creating a pr, the template for the description will autmatically appear (I will add this once we come up with a template). Please follow the instructions on the template.
+
+* Rebase when possible, only merge if absolutely necessary.
+    > In order to keep the history of edits in a linear order, try to rebase from the base branch. Rebasing is typically more complicated, and may sometimes be too difficult. If that is the case, then merge.
+
+* Include translated strings with approriate descriptions.
+
+    > Check that you have included the localization/translation for all strings. Make sure the string can be understood by an external translator who may not have any context. If it is ambiguous, add a description. You can learn more about translations [here]()
+
+* Point the base branch to `develop`, unless you are working on smaller pieces of a bigger project (eg. sprints, new feature).
+    > All pr's should have `develop` as the base branch. The exception is when you are creating smaller pieces of a larger feature. If that is the case, the base branch of this larger feature branch should be `develop`. You can learn more about how to change the base branch [here]()
+
+* Take out all unnecessary comments.
+    > Any comments in the code that do not communicate a meaningful message to another developer should be changed to a meaningful message or taken out of the code. GIVE EXAMPLE
+
+* Code has been tested.
+    > Make sure that you have fully tested your code on a device or emulator. It is up to you as the author, to make sure everything is functioning the way that you intended. 
+
+* Be Timely.
+    > After someone has reviewed you PR and left comments, try to address those comments within 48 hours. This keeps it fresh for yourself and the reviewer, making it easier to understand the comments.
```

Comment:
similar [comment](https://github.com/digidem/mapeo-mobile/pull/740/files#r709545097) about using hard numbers for this